### PR TITLE
update MRM Cmdlet doc

### PR DESCRIPTION
**Pull request by zgf**

### CODE REVIEW

**Comment by chrisda** [comment]
@zgf, I'm closing this PR so I can work on it offline. There appears to be a difference in on-premises vs. cloud behavior for the cmdlet, so I want to confirm.

**Comment by zgf** [comment]
@chrisda Got it, Thank you for your help.

File: exchange/exchange-ps/exchange/mailboxes/Get-RecoverableItems.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ This example returns all of the available recoverable deleted messages with the
 ## PARAMETERS
 
 ### -Identity
-The Identity parameter specifies the mailbox that contains the Recoverable Items folder that you want to view. You can use any value that uniquely identifies the mailbox. For example:
+The Identity parameter specifies the list of mailbox that contains the Recoverable Items folder that you want to view. You can use any value that uniquely identifies the mailbox. For example:
```
Comment (mchiro) [inline exchange/exchange-ps/exchange/mailboxes/Get-RecoverableItems.md]: The Identity parameter specifies the list of mailboxes that contain the Recoverable Items folder that you want to view. You can use any value that uniquely identify mailboxes. For example:

File: exchange/exchange-ps/exchange/mailboxes/Restore-RecoverableItems.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ After using the Get-RecoverableItems cmdlet to verify the existence of the item,
 ## PARAMETERS
 
 ### -Identity
-The Identity parameter specifies the mailbox that contains the Recoverable Items folder where you want to restore items from. You can use any value that uniquely identifies the mailbox. For example:
+The Identity parameter specifies the list of mailbox that contains the Recoverable Items folder where you want to restore items from. You can use any value that uniquely identifies the mailbox. For example:
```
Comment (mchiro) [inline exchange/exchange-ps/exchange/mailboxes/Restore-RecoverableItems.md]: The Identity parameter specifies the list of mailboxes that contain the Recoverable Items folder that you want to restore items from. You can use any value that uniquely identify mailboxes. For example:

File: exchange/exchange-ps/exchange/mailboxes/Restore-RecoverableItems.md [hunk 3]
```diff
@@ -227,7 +227,8 @@ Accept wildcard characters: False
 ```
 
 ### -MaxParallelSize
-The MaxParallelSize paramater controls the maximum number of parallel threads restoring. Higher numbers of parallel threads running will typically increase the speed of Restore-RecoverableItems. Valid values are integers from 1 to 10.
+The MaxParallelSize paramater controls the maximum number of parallel threads restoring. Higher numbers of parallel threads running will typically increase the speed of Restore-RecoverableItems for the multiple mailbox restoring scenario. it cannot work on single mailbox restoring scenario. Valid values are integers from 1 to 10.
```
Comment (mchiro) [inline exchange/exchange-ps/exchange/mailboxes/Restore-RecoverableItems.md]: The MaxParallelSize parameter controls the maximum number of restoring threads in parallel. For multiple mailbox restore scenarios, running more threads in parallel will typically improve the completion time of Restore-RecoverableItems. Parallel threads will not improve single mailbox restore scenarios. 

Valid values: integers from 1 to 10.

**Comment by chrisda** [comment]
@zgf, I already made similar updates, but I forgot to merge them yesterday

**Pull request closed without merging**